### DISCUSSION TITLE
How to use stackStripMap.py to process ERS and Envisat imagery?

### DISCUSSION THREAD

**Discussion by leeyhgeo** [isce-framework/isce2#653] [Q&A]
I'm wondering how to process ERS and Envisat imagery with stackStripMap.py. I have collected some ERS (.E1 & .E2) and Envisat (.N1) images. What's the standard workflow for processing images of these two satellites?

**Comment by Alquimista0830** [comment]
Hi! Did you figure out the solution to this issue? I encountered the same problem as yours, and am searching the discussion to find the standard workflow but few about this. Appreciated!
